.obs/chartfile/operator/questions.yaml

```diff
@@ -8,13 +8,13 @@ questions:
   group: "Elemental OS Channel"
   subquestions:
   - variable: channel.repository
-    default: registry.suse.com/rancher/elemental-teal-channel
+    default: rancher/elemental-channel
     description: "Specify Elemental OS channel repository"
     type: string
     label: Elemental OS Channel Repository
     group: "Elemental OS Channel"
   - variable: channel.tag
-    default: "1.3.4"
+    default: "%VERSION%"
```

Comment:
This change also requires changes in the `_service` file in OBS. Once this PR gets merged, we should also commit these [changes](https://build.opensuse.org/package/rdiff/isv:Rancher:Elemental:PR:rancher:elemental-operator:PR-584/elemental-operator-helm?linkrev=base&rev=4).